tests/behave/features/environment.py

```diff
@@ -1,11 +1,13 @@
+import os
 from selenium import webdriver
 from selenium.webdriver.firefox.options import Options
 
 
 def before_feature(context, feature):
     options = Options()
-    options.headless = True
+    options.headless = bool(os.environ.get("SELENIUM_HEADLESS", "True") == "True")
```

Comment:
This and the FALA_URL environment variable seems like a separate change? Is there some additional context I'm missing here? If so, probably worth adding something to the commit message or moving to another commit.

Reply:
The FALA_URL is the url that these test would run against. This is set set in the docker-compose.yml file https://github.com/ministryofjustice/laa-cla-e2e-tests/pull/71/files#diff-eb672b1ddd3aa7dae5f2577f38daa271R128

`options.headless = bool(os.environ.get("SELENIUM_HEADLESS", "True") == "True")`
I added this change so that when debugging / writing test locally you can install selenium locally and see the browser interactions